lib/core/src/lib/buttons-menu/buttons-menu.component.spec.ts

```diff
@@ -1,117 +0,0 @@
-/*!
```

Comment:
Why are we removing unit tests instead of moving them?

Reply:
Insights is to be deprecated in the upcoming jiras :)

Reply:
I will port the unit tests, thanks